src/main/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,187 @@
+                        wantToSave();
+                }
+            }
+
+        }
+    }
+
+    private void infoMessage() {
+        System.out.println("Please enter valid command with correct amount parameters.\n"
+                + "You can get list of available commands by entering 'help'.\n");
+    }
+
+    private void create(String path, String filename) {
+        try {
+            Path newFilePath = Paths.get(path + FileSystems.getDefault().getSeparator() + filename);
+            Files.createFile(newFilePath);
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private void read(String path, String filename) {
+        Path filePath = Paths.get(path + FileSystems.getDefault().getSeparator() + filename);
+        try (BufferedReader reader = Files.newBufferedReader(filePath)) {
+            String currentLine = null;
+            while ((currentLine = reader.readLine()) != null) {
+                System.out.println(currentLine);
+            }
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private void info(String path, String filename) {
+        int charNumber = 0;
+        int linesNumber = 0;
+        int wordsNumber = 0;
+        try {
+            Path filePath = Paths.get(path + FileSystems.getDefault().getSeparator() + filename);
+            try (BufferedReader reader = Files.newBufferedReader(filePath)) {
+                String currentLine = null;
+                while ((currentLine = reader.readLine()) != null) {
+                    charNumber += currentLine.length();
+                    wordsNumber += currentLine.split(" ").length;
+                    linesNumber++;
+                }
+            } catch (IOException e) {
+                System.out.println(e.getMessage());
+            }
+            System.out.println("File " + filePath + " has following parameters:\n"
+                    + "Amount of characters: " + charNumber
+                    + "\nNumber of words: " + wordsNumber
+                    + "\nNumber of lines: " + linesNumber
+                    + "\nSize: " + Files.getAttribute(filePath,"size").toString() + " bytes\n"
+                    + "Last modification time: "
+                    + Files.getAttribute(filePath, "lastModifiedTime").toString());
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private void help() {
+        System.out.println("create [path] [filename] - creates new file at path\n"
+                + "read [path] [filename] - reads file at path\n"
+                + "info [path] [filename] - shows info on file at path\n"
+                + "help - shows all commands\n"
+                + "help [command] - shows more detailed info on the command\n"
+                + "exit - exits program");
+    }
+
+    private void help(String command) {
+        switch (command) {
+            case "create":
```

Comment:
better to save all information about comments to the hashmap and get it from here